src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -168,6 +173,14 @@ class IncidentsList extends AsyncComponent<Props, State & AsyncComponent['state'
 }
 
 class IncidentsListContainer extends React.Component<Props> {
+  componentDidMount() {
+    trackAnalyticsEvent({
```

Comment:
if they click on open/resolved/all will this log additional views? can we also record the filter?

Reply:
Updated